main/res/values/dimens.xml

```diff
@@ -42,6 +42,7 @@
 
     <!-- buttons -->
     <dimen name="textSize_buttonsPrimary">@dimen/textSize_detailsSecondary</dimen>
+    <dimen name="textSize_buttonsDouble">@dimen/textSize_detailsPrimary</dimen>
```

Comment:
Is this the same text size as the coordinates currently have in the single button?

Reply:
I increased the size a little bit. Before it was the same as the button text (As "SEARCH BY COORDINATES"). But it was bad to read the coordinate without additional space between the coordinate parts. And we have now more space :-)

Reply:
Sounds good.